src/main/webapp/components/dashboard/dashboard.html

```diff
@@ -0,0 +1,30 @@
+<div class="container-fluid">
+    <div class="row content">
+        <div class="col-sm-3 sidenav">
```

Comment:
508 Accessibility Compliance. use a `<nav>` semantic html5 element to describe your navigation. https://www.w3schools.com/html/html5_semantic_elements.asp